couchdbkit/client.py

```diff
@@ -413,7 +413,7 @@ def open_doc(self, docid, **params):
             doc.fetch()
         except HTTPError as e:
             if e.response.status_code == 404:
-                raise ResourceNotFound
+                raise ResourceNotFound(json.loads(e.response.content)['reason'])
```

Comment:
Is this always specified? Should it use `.get` ?

Reply:
I think it should be, but I'll be on the lookout for any errors